experimental/dds/attributable-map/src/mapKernel.ts

```diff
@@ -361,7 +413,18 @@ export class MapKernel {
 	public getSerializedStorage(serializer: IFluidSerializer): IMapDataObjectSerialized {
 		const serializableMapData: IMapDataObjectSerialized = {};
 		for (const [key, localValue] of this.data.entries()) {
-			serializableMapData[key] = localValue.makeSerialized(serializer, this.handle);
+			const attribution = this.options?.attribution?.track
+				? this.attribution?.get(key)
+				: undefined;
+			serializableMapData[key] = localValue.makeSerialized(
+				serializer,
+				this.handle,
+				attribution && attribution?.type !== "local"
```

Comment:
we should actually assert attribution's type isn't local here. something has gone wrong with summarization if so (summarizing on a client with local edits OR attribution tracking is incorrect), and it's better to fail fast than write incorrect data.